diagnostic.md

```diff
@@ -56,13 +70,29 @@ let answer = 'not empty';
 
 while (answer !== '') {
   answer = ask("Guess my secret? ");
-  if (answer === 'SeCrEt') {
+  if (answer === '7') {
    console.log("You guessed it!");
+   else if (answer <1 && >10){
```

Comment:
Sorry, I made a mistake. That if check should be if (answer < 1 || answer > 10). 

(answer <1 && answer >10) will always be false because there doesn't exist a number that is less than 1 AND greater than 10.